client/components/ThemeContext.jsx

```diff
@@ -0,0 +1,6 @@
+import { createContext } from 'react';
+
+//Initialize ThemeContext
+const ThemeContext = createContext();
+
+export default ThemeContext;
```

Comment:
Don't export default. Please export UserContext and the import in other files should be import {UserContext} from '../path'